apps/dashboard/src/app/projects/projects-list/projects-list.component.html

```diff
@@ -5,17 +5,18 @@ <h1>Projects</h1>
     </mat-card-title>
   </mat-card-header>
   <mat-card-content>
-    <mat-list>
+    <mat-list data-test-id="project-list">
       <mat-list-item
-        *ngFor="let project of projects"
-        (click)="selected.emit(project)">
+        *ngFor="let project of projects; let id=index"
+        (click)="selected.emit(project)"
+        attr.data-test-id="project-{{id}}" [attr.data-title]="project.title" attr.data-cy="{{id}}">
```

Comment:
In general, I wouldn't use `data-cy` unless it was the actual test id I wanted to find.

```suggestion
        attr.data-test-id="project-{{id}}" [attr.data-title]="project.title" attr.data-index="{{id}}">
```